app.js

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env node
+const colors = require('colors');
+const path = require('path');
+const fs = require('fs');
+const markdownLinkExtractor = require('markdown-link-extractor');
+const fetch = require('node-fetch');
+
+let inputDoc = process.argv[2];
+let options = process.argv[3];
+
+const validateArgvExist = (inputFile) => {
```

Comment:
cuando el valor muta se implementa let
```suggestion
let validateArgvExist = (inputFile) => {
```